test/helpers/test-bot.js

```diff
@@ -23,6 +29,11 @@ module.exports = function TestBot (opts = {}) {
   if (opts.name) opts.name = 'ssb-tribes/' + opts.name
 
   const ssb = stack({
+    box2: {
+      legacyMode: true,
```

Comment:
this needs to be documented in README

Reply:
it is in the box2 readme https://github.com/ssbc/ssb-box2/blob/6cef638b50af40ca1dd73c4a734b15dd53ef92d9/README.md#dm-encryption

Reply:
aah you mean that we need to tell users of this that they need the same conf (are we sure they do? probably likely i guess)

Reply:
a bit tempted to set this for the user but i suspect that's illegal :disguised_face:

Reply:
throwing an error now if it's not set, and documented in the readme